nautobot/extras/tests/test_dynamicgroups.py

```diff
@@ -629,6 +630,13 @@ def test_generate_query_for_filter(self):
         serial_qs = Device.objects.filter(serial__iexact=solo_value)
         self.assertQuerySetEqual(solo_qs, serial_qs)
 
+        # Test that a nested field_name w/ `generate_query` works as expected. This is explicitly to
+        # test a regression w/ nested slug-related values such as `DeviceFilterSet.region` which
+        # filters on `site__region`.
+        parent_region = Region.objects.filter(children__isnull=False).first()
+        group.set_filter({"region": [parent_region.slug]})
+        group.validated_save()
```

Comment:
Shouldn't we test `generate_query_for_filter()` on this group?

Reply:
Whoops, I accidentally some lines.